client-index/src/index/default_index.rs

```diff
@@ -469,15 +469,6 @@ mod tests {
         fn broadcast_transaction(&self, _: &[u8]) -> Result<()> {
             Ok(())
         }
-
-        /// Get abci query
-        fn query(&self, _path: &str, _data: &str) -> Result<QueryResult> {
```

Comment:
and deleting your previous PR?

Reply:
oh it not merged with latest.
I'll recheck